src/js/entities-service/actions.js

```diff
@@ -6,11 +6,15 @@ const Entity = BaseEntity.extend({
   radioRequests: {
     'actions:model': 'getModel',
     'actions:collection': 'getCollection',
-    'fetch:actions:model': 'fetchCachedModel',
+    'fetch:actions:model': 'fetchAction',
     'fetch:actions:collection': 'fetchActions',
     'fetch:actions:collection:byPatient': 'fetchActionsByPatient',
     'fetch:actions:collection:byFlow': 'fetchActionsByFlow',
   },
+  fetchAction(id) {
+    const data = { include: 'program-action.program,flow.program-flow.program' };
+    return this.fetchModel(id, { data });
```

Comment:
Would it make sense for this to be ... ?

```suggestion
    const include = [
      'program-action.program',
      'flow.program-flow.program',
    ];
    return this.fetchModel(id, { include: include.join(',') });
```

That way it is easier to add additional includes later?

Reply:
yeah that's reasonable.  I think we used to do that too prior to pulling all of the includes